### PR TITLE
Fix AppVeyor

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -9,6 +9,12 @@ const BinBuild = require('bin-build');
 const compareSize = require('compare-size');
 
 test.cb('rebuild the zopflipng binaries', t => {
+	// Skip the test on Windows
```
Comment (XhmikosR) [inline test/test.js]: Maybe there's a better way to skip the test on Windows? I'm not familiar with ava @sindresorhus

**Comment by sindresorhus** [comment]
Can you use https://github.com/sindresorhus/appveyor-node/blob/master/appveyor.yml instead

**Comment by XhmikosR** [comment]
Sure, but note that your appveyor.yml caches something that doesn't make sense to cache :P

I could make a PR there.

**Comment by sindresorhus** [comment]
> but note that your appveyor.yml caches something that doesn't make sense to cache :P

What doesn't make sense?

File: test/test.js (hunk 1 above)
Comment (sindresorhus) [reply test/test.js]: Yes, this is the best way.

**Comment by XhmikosR** [comment]
It's marked as passed though which isn't true :)

**Comment by XhmikosR** [comment]
Scratch that. It does make sense since you install a new npm globally
regardless of the node version.

I have added caching of local node modules on this PR. This is the only
difference compared to your solution. Let me know if you want me to revert
that.

**Comment by sindresorhus** [comment]
> I have added caching of local node modules on this PR. This is the only
difference compared to your solution. Let me know if you want me to revert
that.

I don't want that. It can (and has) cause lots of weird problems.

**Comment by XhmikosR** [comment]
It **had** weird problems with old npm versions, but your call.

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**